src/index.js

```diff
@@ -11,34 +11,47 @@ import './images/turing-logo.png'
 
 console.log('This is the JavaScript entry file - your code begins here.');
 
-import updateDOM from './domUpdate.js'
-import api from './api.js'
+import Traveler from './Traveler.js';
+
+import updateDOM from './domUpdate.js';
+import api from './api.js';
 
 let userID = Math.floor((Math.random() * 50) + 1);
 let travelers;
+let traveler;
 let trips;
 let destinations;
 
 window.onload = onLoadContent;
+// window.onload = fuck;
+
 
 function onLoadContent() {
-  let promise1 = api.fetchAllTravelers();
-  let promise2 = api.fetchAllTrips();
-  let promise3 = api.fetchAllDestinations();
-  console.log(moment().format('MMMM Do YYYY'));
+  let userID = Math.floor((Math.random() * 50) + 1);
```

Comment:
Noice! I don't think we'll need this eventually though :( For it3 a user will log in with their credentials and that will give us the information to fetchOneTraveler with

Reply:
Yea, I mostly have this bit in for testing purposes. Every time the page loads it boots up with a new user and let's me know that everything is dynamicc